pyanaconda/ui/gui/spokes/custom_storage.py

```diff
@@ -1408,6 +1378,8 @@ def _populate_right_side(self, selector):
 
         # you can't encrypt a btrfs subvolume -- only the volume/container
         # XXX CHECKME: encryption of thin logical volumes is not supported at this time
+        device_type = self._get_current_device_type()
```

Comment:
Maybe device_type could be used also for DEVICE_TYPE_BTRFS comparison a few lines above

Reply:
Fixed in https://github.com/rhinstaller/anaconda/pull/2040/commits/33e22ec648df24b7c7e7a88acc8433eb479acd23.